<!doctype html>
<html lang="en">
  

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="favicon.ico">

    <title>Levy Lab Research</title>

    <!-- Bootstrap core CSS -->
    <link href="https://getbootstrap.com/dist/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="research.css" rel="stylesheet">
</head>

<body>
  <<div class="bg"></div>
  <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
        <a class="navbar-brand" href="index.html">LEVY LAB</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarCollapse">
          <ul class="navbar-nav mr-auto">
            <li class="nav-item">
              <a class="nav-link disabled" href="#">     </a>
            </li>
            <li class="nav-item">
              <a class="nav-link disabled" href="#">     </a>
            </li>
            <li class="nav-item"> 
              <a class="nav-link" href="index.html">Home </a>
            </li>
            <li class="nav-item">
              <a class="nav-link active" href="#">Research<span class="sr-only">(current)</span></a>
            </li>
             <li class="nav-item">
              <a class="nav-link" href="publications.html">Publications</a>
            </li>
              <li class="nav-item">
              <a class="nav-link" href="people.html">People </a>
            </li>
            <li class="nav-item">
              <a class="nav-link disabled" href="meetings.html">Meetings</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="protocols.html">Protocols </a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="contact_us.html">Contact Us </a>
            </li>
            
          </ul>
        </div>
      </nav>
  <br><br>
  <div class="jumbotron-fluid" >
    <h1 class="cover-heading">What we do</h1>
  </div>
  <div class="row justify-content-md-center align-items-center my-md-5 pl-md-5">
      
     <div class="col col-lg-9">
       <p class="lead2">
          Barcode-Sequencing<br>
       </p>  
        <p class="bio">
          A DNA barcode is a string of nucleotides, usually at a defined location in a cell genome, that can represent any feature of that cell, such as a synthetic gene deletion, an engineered exogenous operon, or a naturally-occurring genotype. Because barcodes are usually short, relative abundances of barcoded cell pools can be cheaply assayed using short-read Illumina sequencing. Barcodes can be generated to be distant from each other in sequence space, meaning that sequencing errors do not translate to errors in barcode identification, and barcodes present at frequencies lower than one in a million can be reliably measured. Barcodes are also an important form of biological data compression, whereby much longer pieces of DNA (up to an entire genome) is compressed down to 20-30 nucleotides. Because of these features, barcode-sequencing (bar-seq) has the potential to become a platform technology that can be used to answer many biological questions at extremely high throughput. One major challenge to making this happen is devising tools and methods that allow diverse sets of biological questions to be translated into bar-seq assays. Our lab has developed a "landing pad" a technology thats allow multiple barcodes to inserted in tandem into a common location of a cell's genome and a toolkit by which to <a href="http://msb.embopress.org/content/13/2/913">pair these barcodes with longer DNA elements</a>. We are currently using this platform to study <a href="file:///Users/sasha/Downloads/170589.full.pdf">evolutionary dynamics</a>, <a href="https://www.nature.com/articles/ncomms15586">protein-protein interactions</a>, <a href="http://www.g3journal.org/content/7/1/143.short">genetic interactions </a>, and segregating variation, and we think that these applications only scratch the surface of what be done. A second challenge for advancing bar-seq as a platform technology is in developing quantitative, standardized, and replicable methods that are easily accessible to most labs. Our work in this area is focused on developing <a href="https://academic.oup.com/bioinformatics/article-abstract/34/5/739/4562326?redirectedFrom=fulltext">computational tools for barcode sequencing pipelines</a>, replicable methods of fitness estimation in complex cell pools, standardized primer sets, and spike-in controls that allow for assay performance evaluation. We are embarking on several new projects to explore this topic.  Please contact <a href="contact_us.html">Sasha</a> if you are interested. 
          <br><br>
          
        </p>
        <p class="bio">
          People: <a href="people.html#xianan">Xianan Liu</a>, <a href="people.html#fangfei">Fangfei Li</a>, <a href="people.html#zhimin">Zhimin Liu</a>,  <a href="people.html#adam">Adam Dziulko</a><br>
       </p> 
      </div>
  </div>
 <hr>
 <div class="row justify-content-md-center align-items-center my-md-5 pl-md-5">
      <div class="col-md-auto">
        <img src="evolve_into_barcode.jpg", class="rounded d-block" height="350px">
      </div>
      <div class="col col-lg-6">
       <p class="lead2">
          Evolutionary Dynamics <br>
       </p>  
        <p class="bio">
          Many diseases, such as microbial infections and cancer, are driven by cell populations that evolve within a host. In many cases, these cells are of clonal origin, reach large population sizes (millions to trillions of cells), and compete over short timespans (&lt;1000 generations). Improving our ability to forecast the dynamics of these situations could potentially result in better diagnoses, treatments, and outcomes of disease. To do this, we evolve yeast populations in the lab and combine barcode lineage tracking, sequencing of adaptive clones, and mathematical modelling to analyze evolutionary outcomes. We have found that high resolution lineage tracking can be used to <a href="https://www.nature.com/articles/nature14279">estimate the distribution of beneficial fitness effects</a> and the <a href="https://www.sciencedirect.com/science/article/pii/S0092867416310108">mutational spectrum of adaptation-driving mutations</a>, and that these two measures can be combined to <a href="file:///Users/sasha/Downloads/170589.full.pdf">forecast the early dynamics of adaptive genetic diversity</a>. Current efforts in the lab are focused on understanding how epistasis between adaptive mutations impacts evolutionary trajectories and population dynamics.
          <br><br>
          
        </p>
        <p class="bio">
          People: <a href="people.html#fangfei">Fangfei Li</a>, <a href="people.html#xianan">Xianan Liu</a><br>
       </p> 
      </div>
  </div>
 <hr>
 <div class="row justify-content-md-center align-items-center my-md-5 pl-md-5">
      <div class="col-md-auto">
        <img src="www_network.jpg", class="rounded d-block" width="470px">
      </div>
      <div class="col col-lg-6">
       <p class="lead2">
          Network Dynamics <br>
       </p>  
        <p class="bio">
          Cells are composed of many interacting genes and gene products which ultimately work in concert to determine cell phenotype. There has been a tremendous effort over the past 25 years to fully describe these interaction networks in yeast. For example, the Synthetic Genetic Array and Yeast Two-Hybrid technologies have been used to nearly completely characterize all genetic and protein-protein interactions. These studies have been a cornerstone of molecular systems biology and have contributed insights into the cellular architecture, the biological functions of genes, and the mechanisms of phenotypic robustness. Yet, largely due to the great time and cost required to carry out a comprehensive interactome screen, most interactions have been tested in a single environment and a single genotype, resulting in only a partial view of how these networks behave across the range of naturally occuring genotypes and environments these cells will encounter. Moving beyond this static view of cell networks requires technological advances to increase the throughput and ease of replication in these assays. We have developed a general interaction-Sequencing platform (iSeq), which allows millions of interactions to be screened in parallel using growth of complex cell pools and barcode-sequencing as the output. We are applying this technology to understand how <a href="http://www.g3journal.org/content/7/1/143.full"> genetic interactions</a> and <a href="https://www.nature.com/articles/ncomms15586">protein-protein interactions</a> change across environments and genotypes. We are currently focused on adapting the <a href="http://science.sciencemag.org/content/320/5882/1465.long">Protein Fragment Complementation</a> and CRISPR interference assays for use with our platform.    
          <br><br>
          
        </p>
        <p class="bio">
          People: <a href="people.html#zhimin">Zhimin Liu</a>, <a href="people.html#xianan">Xianan Liu</a>, <a href="people.html#adam">Adam Dziulko</a><br>
       </p> 
      </div>
  </div>
 <hr>
 <div class="row justify-content-md-center align-items-center my-md-5 pl-md-5">
      <div class="col-md-auto">
        <img src="yeast_qg.jpg", class="rounded d-block" width="470px">
      </div>
      <div class="col col-lg-6">
       <p class="lead2">
          High-Throughput Quantitative Genetics <br>
       </p>  
        <p class="bio">
          The field of quantitative genetics focuses on mapping the genetic underpinnings of complex phenotypic traits and has far-reaching applications, such as improving crop and livestock yields, bioproduction, and disease prevention. The most common tools used in the field are Quantitative Trait Locus (QTL) mapping and Genome-Wide Association Mapping Studies (GWAS), which sequence many genomes and associate certain alleles with phenotypic traits. A major challenge is that these studies are generally underpowered and have difficulty identifying all the genetic loci that influence a trait. Even if all loci can be located, these studies are often unable to identify the exact polymorphism within a locus that is responsible the trait, requiring lengthy follow-up experiments. We are collaborating with the lab of <a href="http://www-bcf.usc.edu/~ehrenrei/index.html">Ian Ehrenreich</a> to build next-generation barcode-based tools to improve the power and throughput of these assays. We are using these tools to explore a diverse set of questions related to genetic architecture and evolvability of the cell. We are embarking on several new projects to explore this topic.  Please contact <a href="contact_us.html">Sasha</a>  if you are interesting in discussing this further.    
          <br><br>
          
        </p>
        <p class="bio">
          People: <a href="people.html#xianan">Xianan Liu</a>, <a href="people.html#adam">Adam Dziulko</a><br>
       </p>
      </div>
  </div>

  
  <br /><br />







<footer class="mastfoot mt-auto">
        <div class="inner">
          <p>We are part of <a href="http://jimb.stanford.edu/" style="color:white">JIMB</a>, <a href="https://www.nist.gov/" style="color:white">NIST</a>, and  <a href="http://med.stanford.edu/genetics.html" style="color:white">Stanford University</a>.</p>
        </div>
      </footer>
    </div>
    
  </body>  
     <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script>window.jQuery || document.write('<script src="http://getbootstrap.com/assets/js/vendor/jquery-slim.min.js"><\/script>')</script>
    <script src="https://getbootstrap.com/assets/js/vendor/popper.min.js"></script>
    <script src="https://getbootstrap.com/dist/js/bootstrap.min.js"></script>
    </div>
  </body>
  
</html>